### PR TITLE
[SERV-293] Use Python standard library for file I/O

### PR DESCRIPTION
**Pull request by markmatney**
Confirmed with the reporter that this has fixed the issue (when using a virtual environment, at least).

### CODE REVIEW

File: festerize.py [hunk 1]
```diff
@@ -217,24 +218,37 @@ class FesterizeError(IntEnum):
...
                     )
                 )
-                click.echo(border_char * border_length)
 
                 # Save the result CSV to the output directory.
-                out_file = click.open_file(os.path.join(out, csv_filename), "wb")
-                out_file.write(r.content)
-                out_file.close
+                with open(os.path.join(out, csv_filename), "wb") as f:
+                    num_bytes_written = f.write(r.content)
```
Comment (ksclarke) [inline festerize.py:233]: Good to close the file handle when you're done with it(?)

Comment (markmatney) [reply festerize.py:233]: That's what the `with` construct is for; it's like Java's try-with-resources

Comment (markmatney) [reply festerize.py:233]: More info: https://docs.python.org/3/reference/compound_stmts.html#the-with-statement

Comment (ksclarke) [reply festerize.py:233]: Ah, got it. Thanks.

**Review by ksclarke — APPROVED** [review]

**Review by DRickard — APPROVED** [review]

**Pull request merged**